docs/modules/operation/pages/flows/distributed.adoc

```diff
@@ -1,10 +1,14 @@
 
 [[flows-remote]]
-= Distributed/Remote Flows
+= Distributed/Remote Flows with Minion
```

Comment:
Just as an idea for the title: "Using Minions as a flow collector", the term "flow collector" is well known in this domain.